pkg/acceptance/helpers/security_integration_client.go

```diff
@@ -0,0 +1,72 @@
+package helpers
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/require"
+)
+
+type SecurityIntegrationClient struct {
+	context *TestClientContext
+	ids     *IdsGenerator
+}
+
+func NewSecurityIntegrationClient(context *TestClientContext, idsGenerator *IdsGenerator) *SecurityIntegrationClient {
+	return &SecurityIntegrationClient{
+		context: context,
+		ids:     idsGenerator,
+	}
+}
+
+func (c *SecurityIntegrationClient) client() sdk.SecurityIntegrations {
+	return c.context.client.SecurityIntegrations
+}
+
+func (c *SecurityIntegrationClient) CreateSaml2(t *testing.T, id sdk.AccountObjectIdentifier) (*sdk.SecurityIntegration, func()) {
+	t.Helper()
+	return c.CreateSaml2WithRequest(t, sdk.NewCreateSaml2SecurityIntegrationRequest(id, false, c.ids.RandomSchemaObjectIdentifier().Name(), "https://example.com", "Custom", random.GenerateX509(t)))
```

Comment:
3rd arg of `NewCreateSaml2SecurityIntegrationRequest` is a SAML issuer. Why is it here filled out with c.ids.RandomSchemaObjectIdentifier().Name()?

Reply:
It needs to be unique.

Reply:
But this is not right function to use

Reply:
Fixed.